cekit/templates/template.jinja

```diff
@@ -74,7 +74,8 @@ rm{% for repo in repositories %} /etc/yum.repos.d/{{ repo.filename }}{% endfor %
  #}
 {%- macro repo_clear_cache(pkg_manager) -%}
     {% if pkg_manager in helper.SUPPORTED_PACKAGE_MANAGERS %}
-{{ pkg_manager }} clean all && [ ! -d /var/cache/yum ] || rm -rf /var/cache/yum
+{{ pkg_manager }} clean all \
+        && rm -rf "/var/cache/yum" "/var/lib/rpm" "/var/lib/dnf"
```

Comment:
What about `/var/cache/apt` ? And `/var/cache/dnf` ?

Reply:
I was going to try and avoid Debian stuff for this PR (as there are lots more things to fix there and I'm thinking of working on a larger refactoring of the pkg manager) but I could add the relevant paths here if you wish.

Reply:
microdnf didn't write to `/var/cache/dnf. Perhaps full-fat `dnf` does? I'll check

Reply:
If we are removing clean all then we have to add the apt packages to remove as otherwise this PR will break apt clean.

Reply:
> microdnf didn't write to `/var/cache/dnf`. Perhaps full-fat `dnf` does? I'll check

full DNF does so I'll add that to the list

Reply:
> If we are removing clean all then we have to add the apt packages to remove as otherwise this PR will break apt clean.

Ack, addressed in [9fe526b](https://github.com/cekit/cekit/pull/888/commits/9fe526bddd436241ac49169403774bb4b8b5cbbe).

Reply:
> > microdnf didn't write to `/var/cache/dnf`. Perhaps full-fat `dnf` does? I'll check
> 
> full DNF does so I'll add that to the list

Addressed in [3681255](https://github.com/cekit/cekit/pull/888/commits/36812557cbc98ad31d45d7870e36ed8910ddda70).